client/js/jsonedit.js

```diff
@@ -477,6 +478,19 @@ function jeAddCommands() {
   jeAddNumberCommand('opposite value', '0', x=>-x);
 }
 
+function displayComputeOps() {
+  const keyword = document.getElementById('var_search').value;
+  let results = compute_ops.filter(o => o.name.toLowerCase().includes(keyword.toLowerCase()) || o.desc.toLowerCase().includes(keyword.toLowerCase()));
+  var resultTable = '<table>';
+  if(keyword.length > 0) {
+    for(const r of Object.values(results).sort())
```

Comment:
Does this `sort` do anything? How would it know how to sort objects?

Reply:
Fixed, now it sorts alphabetically